socorro/external/boto/crashstorage.py

```diff
@@ -163,11 +181,11 @@ def save_raw_crash(self, raw_crash, dumps, crash_id):
 
         path = build_keys("raw_crash", crash_id)[0]
         raw_crash_data = dict_to_str(raw_crash).encode("utf-8")
-        self.conn.save_file(path, raw_crash_data)
+        self.save_file(path, raw_crash_data)
```

Comment:
We moved the "ownership" of the bucket to the crash storage, so now it has to pass the bucket along with all the calls. I created `load_file` and `save_file` methods to make it easier.